packages/amazonq/test/unit/codewhisperer/util/editorContext.test.ts

```diff
@@ -115,6 +175,165 @@ describe('editorContext', function () {
         })
     })
 
+    describe('extractSingleCellContext', function () {
```

Comment:
"notebook" is the keyword.

```suggestion
    describe('getNotebookCellContext', function () {
```